src/main/java/core/basesyntax/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,19 @@
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private final Map<String, Integer> fruitStorage;
```

Comment:
The visibility of the 'fruitStorage' should be private to follow the encapsulation principle. It's good practice to limit the access to the class members as much as possible.